### DISCUSSION TITLE
Module "Transmit"?

### DISCUSSION THREAD

**Discussion by Rolf-L** [AlexandreRouma/SDRPlusPlus#633] [Ideas]
is it planned to expand SDR++ with a module, that allows using the Adalm Pluto also for transmitting?

**Comment by tomvdb** [comment]
+1 for adalm pluto transmit

**Comment by 3dfablab** [comment]
would be amazing... hackrf one too ...+++ for your job ( android work well on teclast m40)

**Reply by betzburger** [reply]
+1 for hackrf. At this moment SDRangel seems the only way to do transmit with hackrf.

**Comment by zaxanoid** [comment]
great software..... love to have transmit on the pluto....
